app/src/main/java/com/sopt/gongbaek/di/NetworkModule.kt

```diff
@@ -0,0 +1,101 @@
+package com.sopt.gongbaek.di
+
+import android.content.Context
+import android.net.ConnectivityManager
+import com.jakewharton.retrofit2.converter.kotlinx.serialization.asConverterFactory
+import com.sopt.gongbaek.BuildConfig
+import com.sopt.gongbaek.BuildConfig.DEBUG
+import com.sopt.gongbaek.data.remote.util.NetworkUtil
+import com.sopt.gongbaek.di.qualifier.Auth
+import com.sopt.gongbaek.di.qualifier.GongBaek
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.android.qualifiers.ApplicationContext
+import dagger.hilt.components.SingletonComponent
+import kotlinx.serialization.json.Json
+import okhttp3.Interceptor
+import okhttp3.MediaType.Companion.toMediaType
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import java.util.concurrent.TimeUnit
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+object NetworkModule {
+    @Provides
+    @Singleton
+    fun providesOkHttpClient(
+        loggingInterceptor: HttpLoggingInterceptor,
+        @Auth authInterceptor: Interceptor
+    ): OkHttpClient =
+        OkHttpClient.Builder().apply {
+            connectTimeout(10, TimeUnit.SECONDS)
+            writeTimeout(10, TimeUnit.SECONDS)
+            readTimeout(10, TimeUnit.SECONDS)
```

Comment:
[3]: 개인취향이긴 한데, 타입을 명확하게 표시하는것도 좋은 방법이라 생각합니다~🙇🏻‍♂️
```suggestion
            connectTimeout(10L, TimeUnit.SECONDS)
            writeTimeout(10L, TimeUnit.SECONDS)
            readTimeout(10L, TimeUnit.SECONDS)
```